src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -707,6 +736,13 @@ AdaptiveExecutor(CitusScanState *scanState)
 }
 
 
+static bool
+HasDependedJobs(Job *mainJob)
```

Comment:
Missing comment. What's a depended job?

(minor: this seems like a misspelling of dependentJob, we should probably stop misspelling it)

Reply:
https://github.com/citusdata/citus/pull/3244/files